ssg/entities/profile_base.py

```diff
@@ -245,7 +249,9 @@ def unselect_empty_groups(self, root_group):
         profile_rules = set(self.selected)
         is_empty, empty_groups = self._find_empty_groups(root_group, profile_rules)
         if is_empty:
-            msg = "Profile {0} unselects all groups.".format(self.id_)
+            msg = ("Profile {0} unselects all groups. "
+                   "Check whether it selects any rule or extends any profile."
+                  .format(self.id_))
```

Comment:
please fix indentation

Reply:
Thanks for the review, should be fixed now.